common_module.py

```diff
@@ -80,9 +71,10 @@ def stat_write_lines(self, fout):
             fout.write('Queue: ' + qstat_values['Queue'] + '\n')
             fout.write('...\n')
 
-    def statq_write_lines(self, fout):
-        last_line = self.l.pop()
-        for qstatq_values in self.l:
+    @staticmethod
+    def statq_write_lines(all_qstatq_values, fout):
```

Comment:
purified the function so as to avoid making one class instance for statq_write_lines() and one for stat_write_lines()